workers/social/lib/social/models/computeproviders/stacktemplate.coffee

```diff
@@ -373,9 +373,9 @@ module.exports = class JStackTemplate extends Module
       @update query, (err) =>
 
         return callback err  if err
-        return  unless group.slug
 
-        return callback null, this  if currentAccessLevel is accessLevel
+        if currentAccessLevel is accessLevel or not group.slug
+          return callback null, this
```

Comment:
@gokmen I just thought that shouldn't we throw error when there is no group.slug data provided?
update shouldn't work if we are not able to get group.slug data 😕